velox/type/Type.cpp

```diff
@@ -15,11 +15,15 @@
  */
 
 #include "velox/type/Type.h"
+
+#include <sstream>
```

Comment:
next time, let's put such refactoring in a separate PR to make it easier to review substantive changes